doc/uftrace-recv.md

```diff
@@ -27,6 +27,34 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+This command can be used to receive data from remote.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
+
+
+    # CLIENT :
+    $ uftrace record -H localhost -d example_data --port 1234 example
+
+These options have the following meanings:
+`-H localhost`: The remote host address to receive data.
```

Comment:
What about this?

"Above command sends the trace data (after running the `example` program) to a (remote) machine given by the `-H` option ("localhost" in this case)."